packages/nvidia_nat_rag/src/nat/plugins/rag/rag_functions.py

```
@@ -0,0 +1,213 @@
+            for idx, citation in enumerate(citations.results):
+                doc_name = getattr(citation, "document_name", f"Document {idx + 1}")
+                content = getattr(citation, "content", "")
+                doc_type = getattr(citation, "document_type", "text")
+                description = getattr(citation, "metadata", {}).description
+                print(description)
+
+                results.append(f"**{doc_name}** (type: {doc_type}):\n{description}")
+
+            return "\n\n---\n\n".join(results)
+
+        except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)